kevm-pyk/src/kevm_pyk/kdist/plugin.py

```diff
@@ -27,12 +27,15 @@ def __init__(self, kompile_args: Mapping[str, Any], *, deps: Iterable[str] | Non
     def build(self, output_dir: Path, deps: dict[str, Path], args: dict[str, Any]) -> None:
         verbose = args.get('verbose', False)
         enable_llvm_debug = args.get('enable_llvm_debug', False)
+        debug_build = args.get('debug_build', False)
+        ccopts = args.get('ccopts', [])
 
+        ccopts = ccopts + lib_ccopts(kdist.get('evm-semantics.plugin'), debug_build=debug_build)
```

Comment:
```suggestion
        ccopts = ccopts + lib_ccopts(deps.get('evm-semantics.plugin'), debug_build=debug_build)
```

(Of course the `None` case should be handled.)

Reply:
Done.